smswebapp/apiurls.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
In general, we have the *project* determine the absolute URLs. For example, the ``ui`` application doesn't know that its URLs live under ``ui/``, that is set by the project URLconf. Similarly, the ``api`` application doesn't know that it lives under ``api/``. The ``apiurls.py`` file needs to provide a URLconf with the full absolute paths. Since this requires knowing where the ``api`` module's URLs live in the global hierarchy, it requires knowledge which belongs in the project, not the ``api`` application.